### PR TITLE
Documentation Updated: Installation, landing page and resources

### PR DESCRIPTION
**Pull request by jjfumero**
#### Description

This PR updates the documentation with special attention to the installation steps for all supported OS.

#### Problem description

n/ a.

#### Backend/s tested

Mark the backends affected by this PR.

- [ ] OpenCL
- [ ] PTX
- [ ] SPIRV

#### OS tested

Mark the OS where this PR is tested.

- [ ] Linux
- [ ] OSx
- [ ] Windows

#### Did you check on FPGAs?

If it is applicable, check your changes on FPGAs.

- [ ] Yes
- [X] No

#### How to test the new patch?

```bash
make docs
google-chrome docs/build/html/index.html
```

### CODE REVIEW

File: docs/source/installation.rst [hunk 1]
```diff
@@ -27,68 +27,63 @@ The following table includes the platforms that TornadoVM can be executed.
...
 
 Installation
 *****************
 
 TornadoVM can be built with three compiler backends and is able to generate OpenCL, PTX and SPIR-V code.
-There are two ways to install TornadoVM:
 
-A) Automatic Installation
-===========================
+Installer Script
```
Comment (mairooni) [inline docs/source/installation.rst:55]: Suggestion: Installer Script -> Installation Script

File: docs/source/installation.rst [hunk 2]
```diff
@@ -145,387 +140,113 @@ On Windows by using:
...
+   source setvars.sh
+   make 
 
-   tornado --jvm "-Dtornado.unittests.verbose=True -Xmx6g"  -m  tornado.unittests/uk.ac.manchester.tornado.unittests.tools.TornadoTestRunner --params "uk.ac.manchester.tornado.unittests.arrays.TestArrays"
 
 
 .. _installation_windows:
 
-Installing TornadoVM for Windows 10/11
+Installattion for Windows 10/11
```
Comment (mairooni) [inline docs/source/installation.rst:249]: Installattion -> Installation

File: docs/source/installation.rst [hunk 3]
```diff
@@ -609,11 +330,8 @@ Hereafter, the directory with the source code will be referred as ``<TornadoVM>`
 5. Configure/Compile the TornadoVM Project 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-There are two ways to install it. Choose one of them. We highly recommend the first option, since it will install
-the required software dependencies.
-
 
-A) Using the automatic installation script. This script downloads the following dependencies:
+The installer script downloads the following dependencies:
```
Comment (mairooni) [inline docs/source/installation.rst:334]: If you implement the suggestion above, change this to the "installation script"

File: docs/source/installation.rst [hunk 4]
```diff
@@ -27,68 +27,63 @@ The following table includes the platforms that TornadoVM can be executed.
...
-There are two ways to install TornadoVM:
 
-A) Automatic Installation
-===========================
+Installer Script
+=================
 
-The ``tornadovm-installer`` script provided in this repository will compile/download OpenJDK, CMake, Maven and it will build TornadoVM.
+The ``tornadovm-installer`` script provided in this repository will compile/download ``OpenJDK``, ``CMake``, ``Maven`` and it will build the TornadoVM project.
 This installation script has been tested on Linux, macOS and Windows.
```
Comment (mairooni) [inline docs/source/installation.rst:59]: If you don't apply the suggestion, consider changing the "installation script" to "installer script" for consistency.

File: docs/source/installation.rst [hunk 5]
```diff
@@ -628,12 +346,7 @@ A) Using the automatic installation script. This script downloads the following
    setvars.cmd
 
 
-
-B) Manual installation
-
-In this case, you need to install all dependencies (Java, maven and cmake) before building the project.
-Once all dependencies are provided, execute the following commands:
-
+And TornadoVM is readyt to be used. If you want to recompile with a different backend:
```
Comment (mairooni) [inline docs/source/installation.rst:349]: readyt -> ready

File: docs/source/installation.rst [hunk 6]
```diff
@@ -662,7 +375,7 @@ Once all dependencies are provided, execute the following commands:
 
 .. _installation_mali:
 
-TornadoVM on ARM Mali GPUs
+Installattion for ARM Mali GPUs
```
Comment (mairooni) [inline docs/source/installation.rst:378]: Installattion -> Installation

**Review by mairooni — CHANGES_REQUESTED** [review]

**Review by mairooni — APPROVED** [review]

**Review by stratika — CHANGES_REQUESTED** [review]

File: docs/source/resources.rst [hunk 7]
```diff
@@ -31,6 +40,10 @@ Presentations
 Podcast 
 -------------
 
+-  Adam Bien Podcast with Juan Fumero: `TornadoVM, Paravox.ai: Java, AI, LLMs and Hardware Acceleration <https://airhacks.fm/#episode_282>`_.
+
+-  Adam Bien Podcast with Juan Fumero: `FPGAs, GPUs or Data Science with Java <https://airhacks.fm/#episode_282>`_.
```
[the remark this answers is not in the document]
Comment (jjfumero) [reply docs/source/resources.rst]: I just fixed it.

**Review by stratika — APPROVED** [review]

**Pull request merged**